README.md

```diff
@@ -159,3 +159,20 @@ You can revert this configuration with the following command:
 
 ### Usage
 Once the configuration is done, you just need to use `./mvnw` instead of `mvn` in your commands.
+
+### IIDM version vs PowSyBl core version
```

Comment:
This should be a subsection of `Using Maven Wrapper`:

```suggestion
## IIDM version vs PowSyBl core version
```

Reply:
ok

Reply:
I think it is already because ## Using Maven Wrapper

Reply:
Sorry, I mean "it should **not** be"